splinterd/src/main.rs

```diff
@@ -293,6 +295,13 @@ fn main() {
         .or_else(|| Some("127.0.0.1:8080".to_string()))
         .expect("Must provide a url for REST API endpoint");
 
+    let db_url = matches
```

Comment:
This and the arg stuff should all be behind the 'database' feature flag.

Reply:
updated